.github/r_scripts/run_from_par.R

```diff
@@ -1,4 +1,10 @@
 # code to run the ss_new models
+list.of.packages <- c("parallely", "furrr", "future")
+new.packages <- list.of.packages[!(list.of.packages %in% installed.packages()[,"Package"])]
+if(length(new.packages)>0) install.packages(new.packages)
+
+ncores <- parallelly::availableCores() - 1
```

Comment:
Looks like this could be changed to `parallelly::availableCores(omit = 1)` based on 
https://github.com/HenrikBengtsson/parallelly#availablecores-vs-paralleldetectcores